.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: php
+
+sudo: true
+
+matrix:
+  include:
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+    - php: 7.2
+    - php: 7.3
+    - php: nightly
+
+  allow_failures:
+    - php: nightly
+
+install:
+  - COMPOSER_MEMORY_LIMIT=-1 travis_retry composer install
```

Comment:
Can you also test against lowest dependencies? Check how is done in another php projects on GitHub.

Reply:
Not good idea, if we test agains lowest dependencies, some index name for test will have the same name for example : `TRAVIS_php_helper_7.0_*`. It can result to flacky test.

Reply:
We need to take hand on the api key dealer. @nunomaduro @aseure